build_tools/docker/docker_run.sh

```diff
@@ -75,7 +72,7 @@ function docker_setup() {
     #      whereas the home directory is on the persistent SSD boot disk. It
     #      turns out that makes a huge difference in performance for Bazel
     #      because it is IO bound at 64 cores..
-    local fake_home_dir="${KOKORO_ROOT?}/fake_home"
+    local fake_home_dir="${DOCKER_TMPDIR}/fake_home"
```

Comment:
comment above this could use a tweak

```
    #   2. When running with Kokoro, we mount a local scratch SSD to KOKORO_ROOT
    #      whereas the home directory is on the persistent SSD boot disk. It
    #      turns out that makes a huge difference in performance for Bazel
    #      because it is IO bound at 64 cores..
```

Reply:
I guess? It's still true though and Kokoro is one of the use cases. I can add some "may" language and such, but this already has "when running with Kokoro"

Reply:
Could replace the first sentence with "This allows us to mount a local scratch SSD, separate from a persistent SSD boot disk."

Reply:
Done. I still left a reference to Kokoro. I don't think pretending it doesn't exist is particularly useful and documenting what is going on here is helpful because it required quite a bit of work to figure out.